### DISCUSSION TITLE
Card error. Please make sure you have the correct card information or try another card.

### DISCUSSION THREAD

**Discussion by StartupYard2** [Glench/ExtPay#200] [Q&A]
The Extension is not on the Chrome store yet thus I am in test mode but I want to test the creation of a paying new user (not just me as a developer).
<img width="464" alt="Screenshot 2024-03-12 at 11 45 36" src="https://github.com/Glench/ExtPay/assets/68431823/386a70f4-9a75-4098-9f9c-ca75f7366c4b">

I tried with 3 different credit cards (with Stripe in Live and Test mode) but always get this same error. How do I debug this?

Or ExtensionPay in test mode does not allow real transactions? I could not find this info in the doc. 

Any help is welcome. Thanks!

**Comment by Glench** [comment]
ExtensionPay in dev mode does not accept payments, no, but you also need to make sure your extension's price when converted into your local currency is more than 50 cents.
